Documentation/SwiftGenKit Contexts/files.md

```diff
@@ -0,0 +1,60 @@
+# Files parser
+
+## Input
+
+The files parser accepts multiple files or directories, which it will recursively search for files that match the given filter (default `.*`).
+
+The list of files will be converted into a structured directory tree.
+
+## Output
+
+The output context has the following structure:
+
+-`files`:`Array` — list of files in the base directory specified
+-`dirs`:`Dictionary` — tree structure of subdirectory and its contents
```

Comment:
Seems to me that this is not a dictionary but more an array of dictionaries, right?
(With each Dictionary/entry having the keys `name`, `files` and `dirs`)

(Actually you properly documented it as an `Array` for the recursive entry on line 17)